web-app/src/components/Header.js

```diff
@@ -18,13 +20,38 @@ const useStyles = makeStyles((theme) => ({
 
 export default function Header() {
 	const classes = useStyles();
+	const [state, setState] = useState({
+		checkedA: true,
+	});
+	const handleChange = (event) => {
+		setState({ ...state, [event.target.name]: event.target.checked });
+	};
+
 	return (
 		<div className={classes.root}>
 			<AppBar position="static">
 				<Toolbar>
 					<Typography variant="h6" className={classes.title}>
 						Vachan Admin
 					</Typography>
+					<Typography component="div" className={classes.title}>
+						<Grid
+							component="label"
+							container
+							alignItems="center"
+							spacing={1}
+						>
+							<Grid item>Staging</Grid>
+							<Grid item>
+								<Switch
+									checked={state.checkedC}
```

Comment:
You can add color="primary" to the switch to make it blue instead of the default red